extensions/guacamole-auth-jdbc/modules/guacamole-auth-jdbc-base/src/main/java/org/apache/guacamole/auth/jdbc/connection/ConnectionModel.java

```diff
@@ -165,6 +172,31 @@ public Integer getMaxConnectionsPerUser() {
     }
 
     /**
+     * Sets the connection weight.
+     *
+     * @param connectionWeight
+     *     The weight of the connection.  null is acceptable, negative values
+     *     indicate that the connection should not be used.
+     */
+    public void setConnectionWeight(Integer connectionWeight) {
+        this.connectionWeight = connectionWeight;
+    }
+
+    /**
+     * Returns the connection weight used in calculating the
+     * weighted algorithms.
+     *
+     * @return
+     *     The connection weight as an int.  If the weight is
```

Comment:
I'm guessing this is from the previous version of this function? It currently just returns the connection weight, with the handling of default value, null, etc. being (correctly) elsewhere.